pkg/servicescm/servicescm.go

```diff
@@ -95,16 +98,20 @@ type Data struct {
 	Files []FileInfo `json:"files"`
 	// EnvironmentVars will be set as system-level environment variables on each Windows instance
 	EnvironmentVars map[string]string `json:"environmentVars,omitempty"`
+	// SupportedEnvVars contains information about the WMCO supported environment variables
+	SupportedEnvVars []string `json:"supportedEnvVars,omitempty"`
```

Comment:
In general I'm not a huge fan of the `supportedEnvVars` naming, specifically `supported`.
From a technical sense `watchedEnvironmentVariables` would be more fitting.
`support` is more of a business term, while what this api is really doing is directing the WICD controller to ensure the state of specific resources.

Reply:
I don't have strong opinion on this. It made sense to me as WICD supports these set of env Vars. I can update it.